test/include/dlaf_test/matrix/util_matrix_blas.h

```diff
@@ -0,0 +1,60 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2019, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+/// @file
+
+#include "blas.hh"
+#include "dlaf/matrix.h"
+#include "dlaf_test/matrix/util_matrix.h"
+#include "dlaf_test/util_types.h"
+
+namespace dlaf {
+namespace matrix {
+namespace test {
+using namespace dlaf_test;
```

Comment:
Do we still need the `using namespace dlaf_test` here?

Reply:
For the moment yes, because of `TypeUtilities`.

I've opened issue #139, we will fix all this kind of things once a ruleset will be decided.